source/clear-linux/concepts/swupd-about.rst

```diff
@@ -3,43 +3,30 @@
 Software update
 ###############
 
-The traditional way
-===================
+Clear Linux OS for Intel® Architecture does software updates differently than traditional Linux-based operating systems. Where traditional distributions rely on packages for software deployment, Clear Linux uses the concept of a bundle for deployment. Traditional Linux packages provide a particular utility or library, whereas Clear Linux bundles provide a specific function and include all necessary dependencies and packages.
```

Comment:
@krisdale. Very nice synthesis of the first two paragraphs.

Reply:
Minor edits in line 6: 

Make sure each new line has a hard return (so it shows a line no.)

Use "|CLOSIA|" in place of "Clear Linux for  Intel® Architecture."  

"Whereas traditional distributions [...] , `Clear Linux\*` uses bundles for deployment."

Reply:
@mvincerx Thanks for the syntax corrections!